### DISCUSSION TITLE
Monster Hunter: Rise

### DISCUSSION THREAD

**Discussion by Naccal** [Haato3o/HunterPie#179] [General]
### MH Rise

Of course I understand the game just hit the PC release, but are there plans to make HunterPie compatible with Monster Hunter Rise?

**Comment by haato3o** [comment]
I have plans to support Monster Hunter: Rise in this other [project](https://github.com/Haato3o/HunterPie-v2/). 

https://github.com/Haato3o/HunterPie-v2/issues/20

**Reply by Naccal** [reply]
Oh wow, thank you so much for getting back to me about this, I'm so silly for not clicking your name to check out other projects.
This makes me so happy to hear your putting so much love into your project.

I hope when you publish a release I'll be there to help, give feedback and of course, play with HunterPie more!
To me its an essential part of Monster Hunter.
